extras/indexingExample/src/main/assembly/assembly.xml

```diff
@@ -33,6 +33,7 @@ under the License.
             <includes>
                 <include>org.apache.rya:rya.indexing:*:accumulo-server</include>
                 <include>org.locationtech.geomesa:geomesa-accumulo-distributed-runtime:*</include>
+                <include>mil.nga.giat:geowave-datastore-accumulo:*</include>
```

Comment:
I added each geowave dependency individually.  The accumulo deploy dependency isn't available on the maven repo.  We'd have to build it if we wanted to use it.  The deploy pom is here: https://github.com/ngageoint/geowave/blob/9d6cfa9fb40e6e17650df3b41854d91402a91648/deploy/pom.xml